### ISSUE TITLE
Unable to install interop Python module in Docker container.

### ISSUE THREAD

**Issue by lutrarutra** [Illumina/interop#350]
I am unable to install interop in a docker container. I have tried versions 3.9-3.12, as well as ubuntu:latest.

Steps to reproduce:
```bash
docker run -it python:3.10
# -> attach to shell
apt update
pip install numpy     # OK
pip install interop   # Error
ERROR: Could not find a version that satisfies the requirement interop (from versions: none)
ERROR: No matching distribution found for interop
```

**Comment by ezralanglois** [comment]
I cannot reproduce this issue.

I am running:
Docker version 19.03.12, build 48a66213fe

So I need to add `--security-opt seccomp=unconfined`

```
$ sudo docker run  --security-opt seccomp=unconfined -i -t --rm python:3.10 bash
root@39777df44b2c:/# pip install interop
Collecting interop
  Downloading interop-1.3.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.4/4.4 MB 7.8 MB/s eta 0:00:00
Collecting numpy>=1.21.6
  Downloading numpy-2.1.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (16.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 16.3/16.3 MB 56.4 MB/s eta 0:00:00
Installing collected packages: numpy, interop
Successfully installed interop-1.3.2 numpy-2.1.1
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

[notice] A new release of pip is available: 23.0.1 -> 24.2
[notice] To update, run: pip install --upgrade pip
root@39777df44b2c:/# 
```

**Comment by lutrarutra** [comment]
Seems to be issue with host platform(M1 Apple Silicon/ARM); After adding `--platform=linux/x86_64`, I was able to install Interop in docker:

```
❯ docker run --platform=linux/x86_64 -i -t --rm python:3.10 bash
root@6e17ac17890e:/# pip install interop
...
Installing collected packages: numpy, interop
Successfully installed interop-1.3.2 numpy-2.1.1
```
